source/tutorial/triggers-fts.txt

```diff
@@ -0,0 +1,365 @@
+.. _triggers-fts-tutorial:
+
+====================================================
+Build an Application Using Triggers and Atlas Search
+====================================================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+You can use Atlas App Services with :ref:`Atlas Triggers <triggers>` and 
+:ref:`Atlas Search <fts-top-ref>` to build features on top of your 
+applications.
+
+In this tutorial, you start with a pre-built TypeScript mobile application 
+that includes a working React Native application (frontend) and its corresponding 
+App Services App configuration files (backend). This template app is a basic 
+to-do list application that lets users do various things to manage their tasks.
+To learn more about the template app, see :ref:`react-native-tutorial`.
+
+After you've got the template app running, you will add a new feature that alerts you 
+when a user creates a task that contains a specific word or phrase. The feature 
```

Comment:
Confirming that this template app allows multiple users to create tasks, correct? Asking because I'd like to make sure that the _use case_ is clear here, ie. User A gets notified if User B adds a task with the term "urgent" in it. It may also help to give an example of what type of alert terms users might find to be useful at the beginning of the tutorial instead of only at the end.

Reply:
Yes, that's correct! This is a great piece of feedback - will include an example in the intro now